app/controllers/MissionController.scala

```diff
@@ -66,7 +81,12 @@ class MissionController @Inject() (implicit val env: Environment[User, SessionAu
         val concatenated = completedMissionJsonObjects ++ incompleteMissionJsonObjects
         Future.successful(Ok(JsArray(concatenated)))
       case _ =>
```

Comment:
can you just add a comment saying that this is a case for the anon user, now that the code for each has become long enough that it is not immediately obvious?